tests/examples/externals/stencil.py

```diff
@@ -0,0 +1,74 @@
+from dusk.script import *
+
+# integration works naturally through python's import mechanism
+from definitions import *
+
+
+@stencil
+def horizontal_subdomains(edge1: Field[Edge, K]):
+
+    # examples of dusk syntax for horizontal subdomains
+
+    # short + slice syntax
+    with levels_upward, domain[nudging : halo - 1]:
+        edge1 = 1
+        # ...
+
+    # short + list syntax
+    with levels_upward, [nudging, halo - 1]:
+        edge1 = 1
+        # ...
+
+    # long + slice syntax
+    with levels_upward, domain[regions.nudging : regions.halo - 1]:
+        edge1 = 1
+        # ...
+
+    # long + list syntax
+    with levels_upward, [regions.nudging, regions.halo - 1]:
+        edge1 = 1
+        # ...
+
+    # `in` operator + short + slice syntax
+    with levels_upward in domain[nudging : halo - 1]:
```

Comment:
My preference would be to **not** use `in` as it suggests a hierarchical relation between vertical and horizontal, while they are (literally) orthogonal.

Reply:
I'm really unsure about `in`. On one hand, I really don't like it, since python's `in` has a completely different meaning to how it's used here (it checks whether an element occurs inside a collection). On the other hand, it kind of reads quite nicely.